### PR TITLE
[RTM] Update docs

### PR DESCRIPTION
**Pull request by effigies**
This is the start of a doc read-through and update. I haven't yet touched `workflows.rst` and the API docs.

Modifications range from word-wrapping to rephrasing and reorganization. Nothing too dramatic, but some of the information was dated or the wording just felt a little clunky.

@oesteban If you have any text from the upcoming paper you'd like to use, let me know. It probably makes sense to be consistent across media. Speaking of which, we should probably update the package docstrings for [PyPI](https://pypi.python.org/pypi/fmriprep).

### CODE REVIEW

**Comment by oesteban** [comment]
OMG, the package description is so outdated in Pypi!. Sure, I'll have a look around and provide with text from the paper if needed.

File: docs/citing.rst [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+===============
+
+``fmriprep`` is a Nipype_-based pipeline that combines software from several
+neuroimaging packages with custom-written code to flexibly apply
+state-of-the-art preprocessing algorithms to a wide variety of structural and
+functional neuroimages and generate readable reports.
+For a full list of external software and relevant citations (where available)
+of tools and algorithms, please see the `REFERENCES.md
+<https://github.com/poldracklab/fmriprep/tree/master/RERFERENCES.md>`_ table in
+the source distribution.
```
Comment (oesteban) [inline docs/citing.rst]: Can we add the picture of fmriprep poster somewhere in the docs, like I did with mriqc (http://mriqc.rtfd.io)?

https://f1000research.com/posters/6-1129

Comment (effigies) [reply docs/citing.rst]: Maybe in the citation page? Could have a sub-heading like:

```RST
Posters
----------
* OHBM 2017 (`pdf <https://f1000research.com/posters/6-1129>`_)
  [thumbnail linking to full-size PNG]
```

Comment (oesteban) [reply docs/citing.rst]: Cool!

Comment (oesteban) [reply docs/citing.rst]: Added poster in PNG format to docs/

Comment (effigies) [reply docs/citing.rst]: Had added one already. Just going to force push over, unless you're very attached to that png.

Comment (oesteban) [reply docs/citing.rst]: not really :+1:

**Pull request merged**